addons/survey/models/survey_question.py

```diff
@@ -107,8 +107,9 @@ class SurveyQuestion(models.Model):
         help='Labels used for proposed choices: rows of matrix')
     # -- display & timing options
     is_time_limited = fields.Boolean("The question is limited in time",
-        help="Currently only supported for live sessions.")
-    time_limit = fields.Integer("Time limit (seconds)")
+        help="Currently only supported for live sessions.", compute='_compute_time_limit',
+        recursive=True, inverse="_inverse_time_limit", copy=True, store=True)
+    time_limit = fields.Integer("Time limit (seconds)", compute='_compute_time_limit', inverse="_inverse_time_limit", copy=True, store=True)
```

Comment:
Hey :D thanks for checking everything, super appreciate it! :D as for inverse method, I'm using it because we need to be able to change the value time_limit from front_end as well as from back_end. 

From front_end we change this value when we want to make Time limit for a question different from the default time limit, which I don't think we can do without inverse method

Reply:
Oh I see, thanks for info